src/methods.jl

```diff
@@ -1,13 +1,21 @@
 export FiniteDifferenceMethod, fdm, backward_fdm, forward_fdm, central_fdm, extrapolate_fdm
 
 """
-    add_tiny(x::Union{AbstractFloat, Integer})
+    estimate_magitude(f, x)
 
-Add a tiny number, 10^{-40}, to a real floating point number `x`, preserving the type. If
-`x` is an `Integer`, it is promoted to a suitable floating point type.
+Estimate the magnitude of `f` in a neighbourhood of `x`, assuming that the outputs of `f`
```

Comment:
You've said a neighbourhood here and removed type constraints on `x` that previously ensured it was real. This suggests to me that you expect this new method to work for any type where there's some implicit assumptions around `x` being an element of some metric space, however, there aren't tests to that effect and it's not clear to me when you can / cannot expect `one` to be defined. So

1. is my assessment correct?
2. if so, could you add some tests?
3. if not, could you re-introduce the type-constraints and introduce tests for at least `Float64` and `Float32`?

Reply:
Hmm, no, the method is expected to only be run for `x`s that are `AbstractFloat`s. I'll add the type costraint and also add tests for `Float32` and `Float64`.

Reply:
Great, thanks!